test/distributed/optim/test_zero_redundancy_optimizer.py

```diff
@@ -1048,7 +1048,7 @@ def join_process_group(self):
             grads=grads,
         ):
             for _ in range(NUM_EPOCHS):
-                for input in inputs:
+                for _input in inputs:
```

Comment:
Shouldn't input be used here somewhere?

Reply:
The inputs were used at line 972 to collect grads. Here it was used as an index to assert grads.